noobaa_sa/anonymous_account_manager.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import json
+
+from framework.ssh_connection_manager import SSHConnectionManager
+from noobaa_sa.defaults import MANAGE_NSFS
+from noobaa_sa.exceptions import AccountCreationFailed, AccountDeletionFailed
+
+log = logging.getLogger(__name__)
+
+
+class AnonymousAccountManager:
+    """
+    A class to manage the anonymous account in NooBaa
+    """
+
+    NB_CLI_PATH = MANAGE_NSFS
+
+    def __init__(self):
+        self.conn = SSHConnectionManager().connection
+
+    def create(self, uid=None, gid=None, user=None):
+        """
+        Create an anonymous account using the NooBaa CLI
+
+        Args:
+            uid (str|optional): uid of an account with access to the file system
+            gid (str|optional): gid of an account with access to the file system
+            user (str|optional): user name of an account with access to the file system
+
+            Note that either a valid uid and gid pair or a valid user name must be provided
+
+        Raises:
+            AccountCreationFailed: If the creation of the anonymous account fails
+
+        """
+
+        log.info(f"Adding anonymous account: uid: {uid}, gid: {gid}, user: {user}")
+
+        cmd = f"sudo {self.NB_CLI_PATH} account add --anonymous "
```

Comment:
Cant we add support of accepting `--anonymous` flag in existing account.py file?

Reply:
We could move these methods under the `NSFSAccount` class, but I'd probably need to make some adjustments to the class, and I'd rather minimize changes to existing code.

Also, encapsulating this feature specific logic helps keeping both files cleaner and more focused.

Reply:
Addressed in https://github.com/red-hat-storage/noobaa-sa-ci/pull/38/commits/0a358a3b1567b9b87aec00e82293181721f0581b and https://github.com/red-hat-storage/noobaa-sa-ci/pull/38/commits/8fc715e4d238d4530cd51aeaae029e4dbf5f99ad